### PR TITLE
Update ocicrypt to v1.0.3

### PR DESCRIPTION
**Pull request by lumjjb**
Signed-off-by: Brandon Lum <lumjjb@gmail.com>

### CODE REVIEW

**Comment by stefanberger** [comment]
LGTM.

File: vendor/go.mozilla.org/pkcs7/ber.go [hunk 1]
```diff
@@ -205,44 +228,24 @@ func readObject(ber []byte, offset int) (asn1Object, int, error) {
...
 }
 
-func isIndefiniteTermination(ber []byte, offset int) (bool, error) {
-	if len(ber) - offset < 2 {
-		return false, errors.New("ber2der: Invalid BER format")
-	}
-
-	return bytes.Index(ber[offset:], []byte{0x0, 0x0}) == 0, nil
+func debugprint(format string, a ...interface{}) {
+	//fmt.Printf(format, a)
```
Comment (mikebrow) [inline vendor/go.mozilla.org/pkcs7/ber.go:250]: lol

Comment (stefanberger) [reply vendor/go.mozilla.org/pkcs7/ber.go:250]: Well, the debug enablement seems a compile-time feature, so one has to remove the commented code.

Comment (lumjjb) [reply vendor/go.mozilla.org/pkcs7/ber.go:250]: xD

**Pull request merged**